python/pyspark/pandas/indexes/multi.py

```diff
@@ -375,6 +375,13 @@ def name(self) -> Name:
     def name(self, name: Name) -> None:
         raise PandasNotImplementedError(class_name="pd.MultiIndex", property_name="name")
 
+    @property
+    def dtypes(self) -> pd.Series:
```

Comment:
Let's also add it in docs https://github.com/apache/spark/blob/master/python/docs/source/reference/pyspark.pandas/indexing.rst#multiindex-properties.

And add some docstrings

Reply:
Let;s also add versionadded directive in docstring.

Reply:
updated